tests/utils/tensorflow/test_model_data_utils.py

```diff
@@ -177,15 +177,31 @@ def test_extract_features():
 
 
 @pytest.mark.parametrize(
-    "text, intent, entities, attributes",
+    "text, intent, entities, attributes, real_feature_sizes",

```

Comment:
please be careful with names of the variables. These are `sparse_feature_sizes` to be specific.